packages/utils/src/EventHandlers/ConnectorRegistry.ts

```diff
@@ -30,7 +30,7 @@ export class ConnectorRegistry {
       return existingId;
     }
 
-    const newId = shortid();
+    const newId = nanoid();
```

Comment:
Let's rename `getRandomNodeId` to `getRandomId`, then we can import that utility method here too

Reply:
To be able to do that, I moved nanoid to the `@craftjs/utils` package.